crates/api/src/site/registration_applications/approve.rs

```diff
@@ -99,10 +105,17 @@ async fn send_application_approved_email(
 async fn send_application_denied_email(
   user: &LocalUserView,
   settings: &Settings,
+  deny_reason: Option<String>,
 ) -> LemmyResult<()> {
+  let email = &user.local_user.email.clone().expect("email");
   let lang = get_interface_language_from_settings(user);
   let subject = lang.registration_denied_subject(&user.person.name);
-  let body = lang.registration_denied_body(&settings.hostname);
-  send_email_to_user(user, &subject, &body, settings).await;
-  Ok(())
+  let body = match deny_reason {
+    Some(deny_reason) => {
+      let markdown = markdown_to_html(&deny_reason);
+      lang.registration_denied_reason_body(markdown, &settings.hostname)
+    }
+    None => lang.registration_denied_body(&settings.hostname),
+  };
+  send_email(&subject, email, &user.person.name, &body, settings).await
```

Comment:
- send_email is already correct on main branch
- markdown_to_html needs to be added to main branch
- separate strings with/ without deny reason should also be added to main branch

Reply:
We should make an issue or PR for this before merging this one.

Reply:
https://github.com/LemmyNet/lemmy/pull/5653